wrappers/golang/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/pfeairheller/indy-vdr/wrappers/golang
```

Comment:
Should be updated to this hyperledger repo.

Reply:
Good catch, thank you!  Pushing a fix shortly